workflow-templates/devonfw-sync-docs.yml

```diff
@@ -0,0 +1,62 @@
+env:
+  REPO_CONSOLIDATE: "devonfw-guide"
+  BUILD_USER: ${{ secrets.BUILD_USER }}
+  BUILD_USER_PASSWD: ${{ secrets.BUILD_USER_PASSWD }}
+  BUILD_USER_EMAIL:  ${{ secrets.BUILD_USER_EMAIL }}
+on:  
+  push:
+  pull_request:
+    branches:
+      - $default-branch
+name: Generate PDF and Update Website
+jobs:
+  build:
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Doesn't need this properties be declared in variable Syntax starting with dollar?

Reply:
Have you tested it?

Reply:
fixed the expression.

Reply:
> Have you tested it?

this can be tested only it associates with a member repository.